src/main/java/org/fundacion/jala/converter/controller/PythonCompilerController.java

```diff
@@ -44,9 +44,12 @@ public String uploadFile(final @RequestParam("file") MultipartFile file,
     /**
      * Endpoint for proving compile python
      */
-    @GetMapping("/compilePython2")
-    public String uploadFile2() {
+    @PostMapping("/compilePython2")

```

Comment:
I believe it should have a more representative name. Both the method and the mapping path.